src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionPublishButton.tag

```diff
@@ -2,12 +2,8 @@
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
 <%@ attribute name="publishButton" type="teammates.ui.template.FeedbackSessionPublishButton" required="true" %>
 <%@ attribute name="buttonType" required="true" %>
-<a class="btn ${buttonType} btn-tm-actions session-${publishButton.actionNameLowercase}-for-test"
+<a class="session-${publishButton.actionNameLowercase}-for-test"
    href="${publishButton.actionLink}"
-   title="${publishButton.tooltipText}"
-   data-toggle="tooltip"
-   data-placement="top"
-   data-fsname="${publishButton.feedbackSessionName}"
```

Comment:
I removed the tooltip because the options in the other dropdown list have no tooltip. Do you still want me to put tooltip?